README.md

```diff
@@ -126,65 +110,67 @@ Install the required python packages
 python -m pip install -r requirements.txt
 ```
 
-### Telemetry link
 
-**Create a fake Telemetry Gateway**
+## Development
+
 
-Upload `dummy_gateway.ino` to any Arduino board with a USB port. Make sure to uncomment one of these two lines before :
+### Dashboard
 
-```c
-// Uncomment one of these to select the target gateway
-// #define lps
-#define telemetry
+**Run the GUI**
+
+Run `dashboard.py` with a dummy telemetry link
+
+```
+python ./dashboard.py dummy
 ```
 
+Enjoy
 
-**Run the GUI**
 
-Make sure the board is connected to your computer
+### Launch Pad Station control
 
-Make sure `dashboard.py` is set to use the fake gateway
+Get the *Launch Pad Station Board* up and running. See [aesirkth/LaunchPadStation](https://github.com/aesirkth/LaunchPadStation)
 
-```py
-if __name__ == "__main__":
-    # Use this with a RFD900 modem
-    # serial = SerialWrapper(baudrate=115200, name="Telemetry", rfd900=True)
-    # Use this for testing with an Arduino board and `dummy_telemetry.ino`
-    serial = SerialWrapper(baudrate=115200, name="Telemetry", bonjour="TELEMETRY")
-```
 
-Run `dashboard.py`
+**Run the GUI**
+
+Make sure the *Launch Pad Station Board* is connected to your computer
+
+Run `lps_control.py`
 
 ```
-python ./dashboard.py
+python ./lps_control.py
 ```
 
 Enjoy
 
 
-### Launch Pad Station link
+## Flight conditions
+
+
+### Dashboard
 
-**Create a fake LPS Gateway**
+**Run the GUI**
 
-Upload `dummy_gateway.ino` to any Arduino board with a USB port. Make sure to uncomment one of these two lines before :
+Connect a `RFD900` modem to your computer with an FTDI cable
 
-```c
-// Uncomment one of these to select the target gateway
-#define lps
-// #define telemetry
+Run `dashboard.py`
+
+```
+python ./dashboard.py rfd
 ```
 
+Enjoy
 
-**Run the GUI**
 
-Make sure the board is connected to your computer
+### Launch Pad Station control
 
-Make sure `lps_control.py` is set to use the fake gateway
+Get the *Launch Pad Station Board* and the *Launch Pad Station Gateway* up and running. See [aesirkth/LaunchPadStation](https://github.com/aesirkth/LaunchPadStation)
 
-```py
-if __name__ == "__main__":
-    serial = SerialWrapper(baudrate=115200, name="LPS", bonjour="LAUNCHPADSTATION")
-```
+
+**Run the GUI**
+
+Make sure the *Launch Pad Station Gateway* is connected to your computer. Make sure the *Launch Pad Station Board* is powered and within range.
 
 Run `lps_control.py`

```

Comment:
This same command is mentioned twice but the description is a little different.

Reply:
Yes one is for development use and the other for flight conditions use